checklists/apim_checklist.en.json

```diff
@@ -251,6 +251,20 @@
             "severity": "High",
             "link": "https://learn.microsoft.com/azure/azure-resource-manager/management/azure-subscription-service-limits#api-management-limits"
         },
+        {
+            "category": "Management",
+            "subcategory": "Platform Version",
+            "text": "Upgrade the platform version and follow lifecyle. stv1 is retirng on 31 August 2024",
+            "waf": "Reliability",
+            "service": "APIM",
+            "guid": "46f07d33-ef9a-44e8-8f98-67c097c5d8ce",
+            "severity": "High",
+            "graph": 'Resources 
```

Comment:
Not sure if JSON supports multi-line, you might have to consolidate into a single one. Additionally, you need to use single quotes (`'`) instead of double quotes (`"`) inside of the query, otherwise that breaks JSON.

Last but not least, you might want to use the case-insensitive operator (`=~`) instead of the case-sensitive one (`==`) to make your query more resistant to potential future changes in casing.

Reply:
Changed it to single line, and single quotes inside ARG Query. Also changed to case insensitive operator